website/docs/docs/build/udfs.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "User-defined functions"
+description: "Learn how to add user-defined functions (UDFs) to your dbt projects."
+id: "udfs"
+---
+
+# User-defined functions <Lifecycle status="beta" />
+
+<!--Confirm how users can get this feature-->
+:::info Beta feature 
+If you are interested in getting access to this feature, please contact us. 
+:::
```

Comment:
In cloud, it's going to be, available. It won't be behind any feature flag. People can just start using it if they so desire. In OSS they'll have to install a beta version of dbt-core (e.g. `pip install dbt-core 1.11.0b2`). Even though in cloud it isn't behind a feature flag, the feature itself is considered beta at this point in time.